test-case/check-tinyalsa.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+# This script serves as a wrapper to execute a test case script using tinyalsa.
+# It expects the test case script file name (without path) as the first parameter,
+# followed by other parameters required for that test case.
+
+set -e
+
+# Function to check if a command is available
+check_command() {
+    if command -v "$1" &> /dev/null; then
+        echo "$1 is installed."
+    else
+        echo "$1 is not installed. Exiting..."
+        exit 1
+    fi
+}
+
+# Preconditions: Check if tinyalsa executables are present on the DUT
+check_command "sox"
+check_command "tinycap"
+check_command "tinyplay"
```

Comment:
I'm not convinced this wrapper is worth the added lines of code. If one of these commands are missing, `aplay_opts()` or equivalent should already fail with a decent "not found" error message. Is it not already the case? It should be.

Either way this is not strictly required, independent and can be moved to a future PR. This one has already lasted for a very long time :-)